### ISSUE TITLE
back on the block ?

### ISSUE THREAD

**Issue by fredokun** [fredokun/piccolo#8]
Piccolo is back on the block ? That's cool !

**Comment by ad0** [comment]
Yes indeed ! For now I'm just cleaning up code and fixing a few bugs. This way I get my head back into the compiler code so that new features will come soon.
